backend/src/src-vulnerabilities/src/main/java/org/eclipse/sw360/vulnerabilities/VulnerabilityHandler.java

```diff
@@ -129,54 +123,57 @@ public VulnerabilityHandler() throws IOException, SW360Exception {
             return Collections.emptyList();
         }
         Set<String> releaseIds = projectDatabaseHandler.getProjectById(projectId, user).getReleaseIdToUsage().keySet();
-        if (releaseIds == null || releaseIds.size() == 0) {
-            return Collections.emptyList();
-        }
+        return getVulsByReleaseIdWithoutIncorrect(releaseIds, user);
+    }
 
-        return releaseIds.stream()
-                .flatMap(id -> getVulsByReleaseIdWithoutIncorrect(id, user).stream())
+    private List<VulnerabilityDTO> getVulsByReleaseId(Collection<String> releaseIds, User user) {
```

Comment:
As you've changed the signature to accept a `Collection` of ids, it would probably do good to rename the method to `getVulsByReleaseIds` to reflect that.

The same applies to `getVulsByReleaseIdWithoutIncorrect()` and `getReleaseVulnerabilityRelationByReleaseId()`

Reply:
Done.